cmd/kpod/spec.go

```diff
@@ -16,6 +17,22 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func addRlimits(config *createConfig, g *generate.Generator) error {
+	var (
+		ul  *units.Ulimit
+		err error
+	)
+
+	for _, u := range config.resources.ulimit {
+		if ul, err = units.ParseUlimit(u); err != nil {
+			return errors.Errorf("ulimit option %q requires name=SOFT:HARD, failed to be parsed", u)
```

Comment:
Is the error we get back from ParseUlimit() useless or should it be included here (errors.Wrapf()?)

Reply:
Oops, Fixed